tasks/install_dist_pkgs.yml

```diff
@@ -98,3 +149,21 @@
     version: "{{ nrinfragent_choco_version | default(omit) }}"
   when:
     - nrinfragent_os_name == 'windows'
+
+- name: setup agent service
+  service:
+    name: newrelic-infra
+    state: "{{ nrinfragent_service_state }}"
+    enabled: "{{ nrinfragent_service_enabled }}"
+  when:
+    - nrinfragent_state != "absent"
+    - nrinfragent_os_name != 'windows'
+
+- name: setup agent service (windows)
+  win_service:
+    name: newrelic-infra
+    state: "{{ nrinfragent_service_state }}"
+    start_mode: "{{ nrinfragent_service_enabled | ternary('auto','manual') }}"
+  when:
+    - nrinfragent_state != "absent"
+    - nrinfragent_os_name == 'windows'
```

Comment:
@JakubKotkowiak I believe I moved this to a separate YAML - `setup_agent_service.yaml` so this may also be a regression/merge issue.

Reply:
You're completely right, I'll move it where it should be